tests/features/event.feature

```diff
@@ -0,0 +1,260 @@
+@api
+Feature: Event content creation
+  In order to have event on the site
+  As an editor
+  I need to be able to create and see event items
+
+  @javascript
+  Scenario: Make sure that groups of the fields of Event content is configured correctly.
+    Given I am logged in as a user with the "create oe_event content, access content, edit own oe_event content, view published skos concept entities" permission
+    When I visit "the Event creation page"
+    Then I should see "Online"
+    And I should see "Organiser"
+    And I should see "Description"
+    And I should see "Report"
+    And I should see "Registration"
+    And I should not see "Online type"
+    And I should not see "Organiser name"
+    And I should not see "Summary for description"
+    And I should not see "Summary for report"
+    And I should not see "Registration URL"
+
```

Comment:
We should also assert we don't see the fields in the other groups no?

Reply:
But it already check that at least 1 field not visible from each collapsed groups

Reply:
@sergepavle I know but we need to assert them all because they are all hidden in various groups. If we introduce a regression by accidentally moving a field outside the group, the test won't catch it right? The test is for business reqs which say field A,B,C are under group X :)

Reply:
Done